Samples/azure-storage/Azure.Python/setup.py

```diff
@@ -15,7 +15,7 @@
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-REQUIRES = ["msrest>=0.1.0", "msrestazure>=0.1.0"]
```

Comment:
Yeah I think you're right - I probably just ran regenerate:expected:python/azurepython.
I will take care in future to regenerate the samples as well. Thanks!